### PR TITLE
Frosh Week 2023: Updated schedule and embedded video

### PR DESCRIPTION
**Pull request by micahdbak**

### CODE REVIEW

**Comment by micahdbak** [comment]
@modernNeo ready for review :)

File: csss-site/src/events/frosh/templates/frosh/2023/frosh.html [hunk 1]
```diff
@@ -55,6 +55,7 @@
 			<img class="logo" src="{% static 'frosh_static/2023/images/flag_h.png' %}" alt="CSSS RETRO FROSH 2023">
 		</div>
 		<section>
+			<iframe width="640" height="360" src="https://youtube.com/embed/QQ6u_fkrK0I"></iframe>
```
Comment (EarthenSky) [inline csss-site/src/events/frosh/templates/frosh/2023/frosh.html:58]: Consider centering this element in the page. You can do this by adding `display: block; margin: 0 auto;` to the css

Comment (EarthenSky) [reply csss-site/src/events/frosh/templates/frosh/2023/frosh.html:58]: You may also wish to increase the size of the iframe video to something a little larger, such as 720x405

**Review by EarthenSky — COMMENTED** [review]
Looks great aside from the video being uncentered!

**Comment by modernNeo** [comment]
@EarthenSky this Pr look good to you?

**Comment by EarthenSky** [comment]
Yep, looks good to me!

**Pull request merged**